install/0000_30_machine-config-operator_04_deployment.yaml

```diff
@@ -25,6 +25,9 @@ spec:
           requests:
             cpu: 20m
             memory: 50Mi
+        env:
+          - name: RELEASE_VERSION
```

Comment:
And this value gets substituted by `oc adm release new` or so?  `$ git grep RELEASE_VERSION` doesn't give me any hits in origin.

Reply:
Ah, it's substituting the `0.0.1` bit: https://github.com/openshift/origin/commit/6e8a42aef2612db1c6acf494e77a6c96bf2b610d